jest.config.js

```diff
@@ -1,3 +1,5 @@
+const JEST_PROJECT = process.env.JEST_PROJECT
```

Comment:
Is JEST_PROJECT something standard in jest? If it's custom, can we rename it to `TEST_LEVEL`? I find it more easy to understand since it can be related to the test pyramid (unit -> integration -> e2e)

Reply:
I was inspired by https://github.com/facebook/jest/issues/7542#issuecomment-523807670, I think the final implementation in Jest will use the project noun.

Reply:
OK :+1: